models/public/mobilenet-v1-1.0-224/mobilenet-v1-1.0-224.md

```diff
@@ -0,0 +1,63 @@
+# mobilenet-v1-1.0-224
+
+## Use Case and High-Level Description
+
+`mobilenet-v1-1.0-224` is one of [MobileNet V1 architecture](https://arxiv.org/abs/1704.04861) with the width multiplier 1.0 and resolution 224. It is small, low-latency, low-power models parameterized to meet the resource constraints of a variety of use cases. They can be built upon for classification, detection, embeddings and segmentation similar to how other popular large scale models are used.
+
+## Example
+
+## Specification
+
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| Type                            | Classification                            |
+| GFlops                          | 1.148                                     |
+| MParams                         | 4.221                                     |
+| Source framework                | Caffe\*                              |
+
+## Performance
+
+## Input
+
+### Original model
+
+Image, name - `input` , shape - [1x3x224x224], format [BxCxHxW], where:
```

Comment:
in all other docs you specify input/output shape as comma separated list.
```
1,3,224,224
```
in the same time @jkamelin uses [1x3x224x224] as in this file

Could we align shape specification for different models?